File: 01-Introduccion/01_primeraApp/Persona.m
//
//  Persona.m
//  primeraApp
//
//  Created by IVAN MOLERA on 03/11/13.
//  Copyright (c) 2013 UAB. All rights reserved.
//

#import "Persona.h"

@implementation Persona

- (void) caminaAKmPorHora:(CGFloat)paramVelocidadKmPorHora {
    /* Código del método */
}

- (void) correA10KmPorHora {
    /* Llamamos al método desde nuestra propia clase y le pasamos el valor 10 */
    [self caminaAKmPorHora:10.0f];
}

+ (CGFloat) alturaMaximaEnCentimetros {
    return 250.0f;
}

- (void) cantarUnaCancion:(NSString *)paramTitulo volumen:(CGFloat)paramVolumen {
}

@end

File: 01-Introduccion_Swift/Swift_01/Swift_01/Persona.swift
//
//  Persona.swift
//  Swift_01
//
//  Created by IVAN MOLERA on 24/11/15.
//  Copyright © 2015 UAB. All rights reserved.
//

import UIKit

class Persona: NSObject {

    var nombre: String
    var apellidos: String
    var edad: Int
        
    init (nombre:String, apellidos:String, edad:Int) {
        self.nombre = nombre
        self.apellidos = apellidos
        self.edad = edad
    }
    
    func nombreCompleto() -> String {

        return nombre + " " + apellidos
    }
}
